Backend/api/views/address_view.py

```diff
@@ -1,54 +1,110 @@
 from rest_framework import status
-from rest_framework.permissions import IsAuthenticated
+from rest_framework.permissions import IsAuthenticated, AllowAny
 from api.serializers.address import AddressSerializer
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from api.models.address import Addresses
 from django.contrib.auth.models import User
+from requests import Session
 from api.utils import generateredeemscript, generateservicekey
 
 
 class GenerateAddress(APIView):
     """This view generates address from the user keys and service key"""
-    
-    
+
     permission_classes = (IsAuthenticated,)
-    
-    def post(self,request):
+
+    def post(self, request):
         key1 = request.data["key1"]
         key2 = request.data["key2"]
-        user_id= request.user.id
-        user=User.objects.get(id=user_id)
+        user_id = request.user.id
+        user = User.objects.get(id=user_id)
 
         print(user_id)
         # user=User.objects.get(id=user_id)
-        service_key=generateservicekey()
-        redeem= generateredeemscript(key1,key2,service_key)
-        addr=redeem.address(network='testnet')
-        address=Addresses(address_generated=addr,user_id=user,redeem_script=redeem,service_key=service_key)
+        service_key = generateservicekey()
+        redeem = generateredeemscript(key1, key2, service_key)
+        addr = redeem.address(network="testnet")
+        address = Addresses(
+            address_generated=addr,
+            user_id=user,
+            redeem_script=redeem,
+            service_key=service_key,
+        )
         address.save()
         return Response(
-            {"status":status.HTTP_201_CREATED,"address":address.address_generated}
+            {"status": status.HTTP_201_CREATED, "address": address.address_generated}
         )
 
 
-class GetAddressInfo(APIView):
+class ImportAddress(APIView):
```

Comment:
Should this not be export address rather than import

Reply:
Oh, my bad. I will change it